user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Dashboards_app/Tutorials/Embed_Dashboard_in_Visio.md

```diff
@@ -0,0 +1,143 @@
+
+Expected duration: 20 minutes
+
+> [!TIP]
+>
+> - An application package is available in the [Catalog](https://catalog.dataminer.services/details/package/5182) that demonstrates the outcome of this tutorial.
+> - When you design a dynamic dashboard (e.g. with a configurable element), you need to use a [feed](xref:Feeds), which can be a [Dropdown](xref:DashboardDropdown) component, so that a value can be dynamically assigned when the dashboard is embedded.
+
+> [!NOTE]
+> The content and screenshots for this tutorial have been created in DataMiner 10.3.7.
+
+## Prerequisites
+
+- Select a suitable element in your DataMiner System for which you will be able to embed a dashboard in Visual Overview as illustrated below. If no such element exists yet, create one.
+
+- DataMiner 10.2.0/10.2.2 or higher.
+
+- Microsoft Visio 2010 (with Visio 2010 SP2 or the Visio Compatibility Pack) or a more recent version of Microsoft Visio.
+
+## Overview
+
+The tutorial consists of the following steps:
+
+- [Step 1: Create a dynamic dashboard](#step-1-create-a-dynamic-dashboard)
+- [Step 2: Prepare the URL for use in Visio](#step-2-prepare-the-url-for-use-in-visio)
+- [Step 3: Create Visio and embed Dashboard](#step-3-create-a-visio-file-and-embed-the-dashboard)
+
+## Step 1: Create a dynamic dashboard
+
+1. [Create a new dashboard](xref:Creating_a_completely_new_dashboard).
+
+1. Add a [Dropdown](xref:DashboardDropdown) component to show all the elements of a specific connector:
+
+   1. In edit mode, drag and drop the *Dropdown* visualization from the pane on the left on to the main dashboard area.
+
+   1. In the data pane, select *Elements* under *All available data*, and drag this to the component.
+
+      The dropdown component is now configured to show all available elements.
+
+   1. Click the yellow filter icon under the component, select a protocol under *All available data* > *Protocols*, and drag it to the component.
+
+      The dropdown component will now only show elements using this specific protocol. Make sure this is the protocol of the element you have selected to use for this tutorial.
+
+1. Add extra components to your dashboard:
+
+   1. Add a *State* visualization from the pane on the left, and configure it to use the data *Feeds* > *Dropdown* > *Elements*.
+
+      This component will show the name of the selected element.
+
+   1. Add another *State* visualization, configure it to show one or more parameters of interest based on the *Parameters* > *From: Protocol* data section, and use the yellow filter icon to add the dropdown component you made earlier as a filter.
```

Comment:
Might be better to say that you add the selected element of the dropdown as a filter to the new state (and not the dropdown component).

Reply:
Since I want to keep it clear what should be selected in the data pane, OK if I adjust this as follows?

```suggestion
   1. Add another *State* visualization, configure it to show one or more parameters of interest based on the *Parameters* > *From: Protocol* data section, and use the yellow filter icon to add *Feeds* > *Dropdown* > *Elements* as a filter.
```

I could then also change the line below this to say "This component will show the name and value of the parameters you have added for the element selected in the dropdown component."

Reply:
Yes, definitely an improvement!